### DISCUSSION TITLE
Welcome to reliese/laravel Discussions!

### DISCUSSION THREAD

**Discussion by CristianLlanos** [reliese/laravel#187] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by firdausg** [comment]
Hi, 

Your plugin is one of the best and i recommended for. Since im a kind of user who design database and implement the first process of development by creating DB, your plugin so much helpful. 

Im facing a problem now, which my database server were corrupted. Since we are in development phases, no backup etc, but we do create models using your plugin (luckily we did it!!), which mean, our models is coded well by your plugin.. Right now, our problem, is there anyway that i can reverse all model generated by your plugin into table in database server?.. plz help me, im out of option.. TQ in advance...
